frontends/web/src/routes/account/account.tsx

```diff
@@ -249,6 +250,7 @@ export function Account({
         </Dialog>
         <Header
           title={<h2><span>{account.name}</span>{insured && (<Insured/>)}</h2>}>
+          <HideAmountsButton />
```

Comment:
Another conflict was here, between the line with `insured` logic and badge and `HideAmountsButton` component. Again, kept both.